### ISSUE TITLE
training set of object detection containing bboxes of 0 area

### ISSUE THREAD

**Issue by zhoujuncc1** [ContinualAI/clvision-challenge-2022#3]
There are some instances in the object detection training set containing bboxes of 0 area, e.g.
```{"id": 127172, "image_id": 569716681169025, "bbox": [940.79, 197.9, 77.73, 0], "category_id": 351, "instance_id": "60E87098DACA63C09F51E202A6B813E3_351_0", "area": 0.0}```
Is this a bug or intended behavior?

Thanks.

**Comment by lrzpellegrini** [comment]
Hi Zhou, thank you for reporting this. That's strange. I'll investigate it ASAP.

**Comment by Millielele** [comment]
> 569716681169025

@lrzpellegrini  hello, I also find this problem.So how should i solve that?

**Comment by lrzpellegrini** [comment]
There seems to be a single box with an area <= 0.0, which is that one. It seems to be connected to the mug over the fridge, so it's definitely an issue with the annotation. Fortunately, that annotation is not a main object (the main object for that image is the refrigerator), so this only affects the "Category Detection" track. In addition, the test set contains no such problems, so there are no issues on the scoring side.

Let me see if I can somewhat fix this for everyone without re-creating the JSON!

**Comment by lrzpellegrini** [comment]
Solved in the main branch. Closing!
